Formula/mopidy-somafm.rb

```diff
@@ -2,18 +2,18 @@ class MopidySomafm < Formula
   desc "Mopidy extension for playing music from SomaFM"
   homepage "https://github.com/AlexandrePTJ/mopidy-somafm"
   url "https://files.pythonhosted.org/packages/0e/6a/4cc6c5d1c813ec98343f24be7df3318acf118d2920ceeea6b6d31da1f1f0/Mopidy-SomaFM-2.0.2.tar.gz"
-  sha256 "0c2d1e9b192859f8c61e28760bc9c8341141c9fe76d577fedcab38251c4d3cb3"
+  sha256 "0c2d1e9b192859f8c61e28760bc9c83.1241c9fe76d577fedcab38251c4d3cb3"
```

Comment:
The new dot in the hash here is probably not intentional.

Reply:
Yeah, thanks for pointing that out